envi/cli.py

```diff
@@ -476,6 +478,17 @@ def do_script(self, line):
         locals = self.getExpressionLocals()
         locals['argv'] = argv
 
+        if line.startswith("?"):
```

Comment:
It's probably possible but will require some consideration because the CLI objects are also used to parse " CLI input" from qt widgets in the gui instances.  If there's a way only trigger the readline behavior when it's desired (aka *actually* being used in a terminal on a platform which supports readline) , i would be happy to review and merge it, but be sure to test the delta in the gui case and on systems without readline. ;)